godel_process_execution/CMakeLists.txt

```diff
@@ -1,12 +1,14 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(godel_process_execution)
+add_definitions(-std=c++11)
 
 find_package(catkin REQUIRED COMPONENTS
   abb_file_suite
   godel_msgs
   industrial_robot_simulator_service
   trajectory_msgs
   keyence_experimental
+  swri_profiler
```

Comment:
I was using it to test, but took out the references in the code. I'll remove it.